### PR TITLE
kodi: update to 4c792db

### PR DESCRIPTION
**Pull request by lrusak**
this is needed to put retroplayer stuff in sync

### CODE REVIEW

File: tools/mkpkg/update_binary-addons [hunk 1]
```diff
@@ -89,8 +89,7 @@ validate_pkg_url() {
 }
 
 if [ ! -d $KODI_DIR ] ; then
-  echo "meh.. $KODI_DIR does not exist"
-  exit 127
+  git_clone https://github.com/xbmc/xbmc master $KODI_DIR
```
Comment (MilhouseVH) [inline tools/mkpkg/update_binary-addons]: Should this be `$1` instead of `master`?

Comment (MilhouseVH) [inline tools/mkpkg/update_binary-addons]: Hmm actually the use of `master` everywhere makes me wonder why we pass a <branch-name> parameter at all.

Comment (lrusak) [inline tools/mkpkg/update_binary-addons]: let me fix that

**Comment by lrusak** [comment]
fixed

**Pull request merged**